packages/p2p-v2/package.json

```diff
@@ -12,6 +12,7 @@
         "start": "rimraf dist && npm run test && npm run serve"
     },
     "dependencies": {
+        "@deriv-com/ui": "0.0.1-beta.4",
```

Comment:
can ya upgrade to the 0.0.01-beta.5 version? do you not have any issues with the components saying its not a jsx element?